compile-solidity.sh

```diff
@@ -3,7 +3,7 @@
 # TODO find a way to specify node_modules as an additional source root for solc
 # TODO or migrate to truffle and the entire compilation workflow will be changed
 ENS=$(readlink -f ./node_modules/@ensdomains)
-OZS=$(readlink -f ./node_modules/@ensdomains/ethregistrar/node_modules/openzeppelin-solidity)
```

Comment:
Originally we had a copypasted openzeppelin contract among our contracts. Then I replaced it with a npm dependency, which got installed into node_modules root. Ethregistrar however also depends on openzeppelin contracts and since it didn't like the one I had we got another openzeppelin in buried in `ethregistrar/node_modules`. These nuances were however lost on our compilation workflow, which only uses one location for OZS. When I noticed, I directed it to the buried OZS folder, since that's the one important for ethregistrar (NMBZ contracts are fine with either).

That was all good until I cleaned and reinstalled node_modules. On a clean install, since we no longer directly require any version of OZS, the OZS required by ethregistrar no longer gets installed into the buried ethregistrar folder, but to node_modules root instead. So that's the correct location, the old one wouldn't work on a clean npm install.

Reply:
I am not sure this works like you described. I did `lein npm i` and without the open zepelin dependency some of the functionality stops working.

Reply:
Please check with `rm -rf node_modules` and `lein npm install`.

> I also rebased your PR upon my tests and found out that you need to add [openzeppelin-solidity "2.3.0"] back to project.
Otherwise the FE tests pass :)

:+1: but OZS is specified in `node_modules/@ensdomains/ethregistrar/package.json` and shouldn't be anywhere else.

Reply:
For benefit of future readers, `package-lock.json` was needed. Leiningen deletes `package.json` itself but keeps the lock file for management.